src/config.rkt

```diff
@@ -88,6 +88,7 @@
 (define *starting-prec* (make-parameter 256))
 (define *max-mpfr-prec* (make-parameter 10000))
 (define *ground-truth-extra-bits* (make-parameter 20))
+(define *use-mixed-sampling* (make-parameter #f))
```

Comment:
You could also make this a config flag. Basically, add something like `mixed` under `setup` to `all-flags` and `default-flags` (if you want it on by default) and then check if it's set with `(flag-set? 'setup 'mixed)`. The advantage of this mechanism is that you can set the flag on the command line & it'll work correctly with multiple threads.